src/dotnet/CarbonAware.Aggregators/src/SciScore/SciScoreAggregator.cs

```diff
@@ -0,0 +1,72 @@
+using CarbonAware.Model;
+using CarbonAware.Interfaces;
+using Microsoft.Extensions.Logging;
+using System.Collections;
+using System.Globalization;
+
+namespace CarbonAware.Aggregators.SciScore
+{
+    public class SciScoreAggregator : ISciScoreAggregator
+    {
+        private readonly ILogger<SciScoreAggregator> _logger;
+
+        private readonly ICarbonIntensityDataSource _carbonIntensityDataSource;
+
+        public SciScoreAggregator(ILogger<SciScoreAggregator> logger, ICarbonIntensityDataSource carbonIntensityDataSource)
```

Comment:
I would think about adding xml docs on public level constructors if you haven't already (I don't see an inherit doc here but I could be missing something)